crates/rolldown_debug_action/src/definitions/module_graph_ready.rs

```diff
@@ -11,6 +11,26 @@ pub struct ModuleGraphReady {
 pub struct Module {
   pub id: String,
   pub is_external: bool,
-  pub imports: Option<Vec<String>>,
+  pub imports: Option<Vec<ModuleImport>>,
   pub importers: Option<Vec<String>>,
 }
+
+#[derive(ts_rs::TS, serde::Serialize)]
+#[ts(export)]
+pub struct ModuleImport {
+  pub id: String,
+  // Refer to crates/rolldown_common/src/types/import_kind.rs
+  /// - `import`: `import { foo } from './lib.js';`
+  /// - `dynamic-import`: `import('./lib.js')`
+  /// - `require-call`: `require('./lib.js')`
+  /// - `import-rule`: `@import 'lib.js'`
+  /// - `url-token`: `url('./lib.js')`
```

Comment:
Maybe it is better to use `url('./lib.css'` and `@import 'lib.css'`?

Reply:
Yep